test/v1/direct.driver.boltkit.test.js

```diff
@@ -19,30 +19,28 @@
 
 import neo4j from '../../src/v1';
 import {READ, WRITE} from '../../src/v1/driver';
-import boltkit from './boltkit';
-import sharedNeo4j from '../internal/shared-neo4j';
+import boltStub from '../internal/bolt-stub';
 
-describe('direct driver', () => {
+describe('direct driver with stub server', () => {
 
   it('should run query', done => {
-    if (!boltkit.BoltKitSupport) {
+    if (!boltStub.supported) {
```

Comment:
What do you think about switching between describe or xdescribe based on the value of boltStub.supported before L24? Something like

```javascript
const fdescribe = boltStub.supported ? describe : xdesribe;

fdescribe('direct driver with stub server', () => ....
```

Reply:
Interesting point, if will save us from all these `boltStub.supported` checks. Let's maybe do it in a separate PR. I'll create a card.